DOCSUP-4435: Allow nullable key in MergeTree (#17345)
ClickHouse/ClickHouse: docs/en/engines/table-engines/mergetree-family/mergetree.md

ClickHouse does not require a unique primary key. You can insert multiple rows with the same primary key.

> The key in `PRIMARY KEY` and `ORDER BY` can be `Nullable` type. The setting [allow_nullable_key](../../../operations/settings/settings.md#allow-nullable-key) is responsible for supporting this feature.

Reviewer: ```suggestion
You can use columns of the `Nullable` type in  `PRIMARY KEY` and `ORDER BY` clauses. To turn on this feature turn on the [allow_nullable_key](../../../operations/settings/settings.md#allow-nullable-key) setting.
```

> When sorting using an expression `ORDER BY` for `NULL` values, the [NULLS_LAST](../../../sql-reference/statements/select/order-by.md#sorting-of-special-values) principle always works.

Reviewer: ```suggestion
The [NULLS_LAST](../../../sql-reference/statements/select/order-by.md#sorting-of-special-values) principle applies for `NULL` values in the `ORDER BY` clause.
```
